mep/books/templates/books/work_list.html

```diff
@@ -20,8 +20,13 @@
         <label for="{{ form.query.id_for_label }}"></label>
         {% include 'common/form_errors.html' %}
         <fieldset class="group">
-            {% comment %}
             <legend>Filter by:</legend>
+            <div class="inner">
+                <fieldset class="range facet {{ form.circulation_dates.css_classes }}" id="id_{{ form.circulation_dates.name }}">
+                    <legend>{{ form.circulation_dates.label }}</legend>
+                    {{ form.circulation_dates }}
+                </fieldset>
+            {% comment %}
```

Comment:
it looks like there's an extra `<div class="inner">` here. doesn't seem to have affected the look of the page, but thought i would flag

Reply:
whoops, thanks

Reply:
@thatbudakguy this inner div mirrors what we have on the members search page (where I copied it from)